### ISSUE TITLE
Need new module for Lakeshore 370 AC resistance bridge

### ISSUE THREAD

**Issue by pbnjeff89** [masonlab/labdrivers#12]
All the Oxford dry fridges use this to determine temperature, so accessing this would be extremely helpful.

**Comment by hinnefe2** [comment]
It might be worth checking some of the other python lab driver projects to see if they already have one:

- https://github.com/ralph-group/pymeasure
- https://github.com/Galvant/InstrumentKit
- https://github.com/QCoDeS/Qcodes

**Comment by pbnjeff89** [comment]
I think I had made a mistake when writing down the number. I couldn't find documentation on the 270 AC bridge, so it's probably a 370.

I did check out pymeasure but InstrumentKit and Qcodes were ones that I had not known about. InstrumentKit seemed to have the file, but it didn't seem complete in any way nor something usable by the masonlab/labdrivers package. It's a good start though, and thanks for the heads up on those projects!
